com.unity.render-pipelines.universal/Shaders/Terrain/TerrainDetailLitPasses.hlsl

```diff
@@ -39,19 +37,23 @@ void InitializeInputData(Varyings input, out InputData inputData)
     inputData.positionCS = input.PositionCS;
     inputData.normalWS = half3(0, 1, 0);
     inputData.viewDirectionWS = half3(0, 0, 1);
-    #if defined(MAIN_LIGHT_CALCULATE_SHADOWS)
-    inputData.shadowCoord = input.ShadowCoords;
+
+    #if defined(REQUIRES_VERTEX_SHADOW_COORD_INTERPOLATOR)
+        inputData.shadowCoord = IN.shadowCoord;
```

Comment:
This should be:
inputData.shadowCoord = input.shadowCoord;

Reply:
Good catch!